client/src/catalog-styles.html

```diff
@@ -89,11 +89,14 @@
         flex: 10 0 270px;
         max-width: calc(100% - 270px);
         min-width: 300px;
+        padding: 0 16px;
```

Comment:
Can we remove this? It affects multiple pages and affects alignment with the top nav.